src/content/reference/k8s-metadata/labels/_index.md

```diff
@@ -0,0 +1,147 @@
+    - Value `flux` on an App resource indicates that the App is reconciled by Flux.
+    - Name of a bundle on an App resource indicates that the App has been installed by a bundle.
+
+### giantswarm.io/monitoring
+
+This label set on the Cluster resource with the value `false` can be used to disable the gathering of Prometheus metrics for the entire cluster, and id prevents deployment of Prometheus for this cluster.
+
+On other resources, the label serves as a discovery mechanism for metrics scraping. We're replacing this mechanism by service monitors.
+
+### giantswarm.io/organization
+
+An organization is a key concept in the Giant Swarm platform, used to isolate [tenants]({{< relref "/vintage/platform-overview/multi-tenancy" >}}). This label is used on a variety of resources to associate them with one organization. The value is the name of the organization.
+
+### giantswarm.io/prevent-deletion
+
+Can be set on certain resource types to prevent deletion. See [Prevent accidental deletion of resources]({{< relref "/vintage/advanced/app-platform/deletion-prevention" >}}).
+
+### giantswarm.io/service-type
+
+- TODO Found on various resources in management and workload clusters.
+- [`fmt`](https://github.com/giantswarm/fmt/blob/278eb4b3318c454e50f24413e7ef2250159f28d6/kubernetes/annotations_and_labels.md?plain=1#L49) has an explanation.
+Value is mostly `managed`
+- also found occurrence `manual` in an [ops recipe](https://github.com/giantswarm/giantswarm/blob/0d16eb4ebb0440608bb1bfd0636d34afa6352cc6/content/docs/support-and-ops/ops-recipes/cilium-rate-limit-issue.md?plain=1#L39).
+- Value `system` is also documented.
+- Example use: [PrometheusRule](https://github.com/giantswarm/prometheus-rules/blob/d440f5cc724d9ad2fe4c7f85c9f0b54090a6858e/helm/prometheus-rules/templates/recording-rules/gs-managed-app-deployment-status.rules.yml#L16) - Here it seems to be used to differentiate workloads managed by Giant Swarm from other workloads.
+
+### giantswarm.io/service-priority
+
+Used on cluster-related resources to indicate the [service priority]({{< relref "/vintage/advanced/cluster-management/labelling-workload-clusters#service-priority" >}}), which means the relative importance in general, of the cluster.
+
+### policy.giantswarm.io/psp-status
+
+- TODO Found on App, Chart, and Cluster resources, MC and WC. On clusters with PSS the value is `disabled`.
+
+### policy.giantswarm.io/resource-kind
+
+- TODO Found in PolicyException and PolicyExceptionDraft resources. Values are like `Deployment`, `CronJob`.
+
+### policy.giantswarm.io/resource-name
+
+- TODO Found in PolicyException and PolicyExceptionDraft resources. Values is the name of the resource the exception is for.
+
+### policy.giantswarm.io/resource-namespace
+
+- TODO Found in PolicyException and PolicyExceptionDraft resources. Values is the namespace of the resource the exception is for.
+
```

Comment:
@giantswarm/team-shield can you provide a good example?

Reply:
This list is not only about labels that customers can add but is also more of an informational source, even for us. So if we use those labels, let's document them :)